man/crystal.1

```diff
@@ -0,0 +1,253 @@
+.\"Crystal Programming Language
+.Dd February 11, 2016
+.Dt CRYSTAL(1) "" "Crystal Programmers Reference Guide"
+.\".Dt CRYSTAL 1
+.Os UNIX
+.Sh NAME
+.Nm crystal
+.Nd Statically type-checked programming language
+.Sh SYNOPSYS
+.Nm
+.Op command
+.Op switches
+.Op program file
+.Op --
+.Op arguments
+.Sh DESCRIPTION
+Crystal is a programming language with the following goals:
+
+.Bl -bullet -offset indent -compact
```

Comment:
Maybe skip the language goals, but quickly explain that `crystal` is the compiler for the Crystal language and stick to usage: explaining why `--` is required to pass arguments to the program in the `run` command, that the `run` command can just be skipped, or that one should use the `--release` flag to turn LLVM optimizations on for distributing a binary or running benchmarks.